circuit_parameters_gen/calc_round_numbers.py

```diff
@@ -0,0 +1,120 @@
+# Credit: https://github.com/daira/pasta-hadeshash/blob/master/code/calc_round_numbers.py
+from math import *
```

Comment:
I named it this way because there's also the `N_ASSETS` for the circuit in the `src/merkle_sum_tree/params.rs`, so it's not just for Poseidon. Do you still think we should rename it?

Reply:
I understand, makes sense. Let's keep it as it is! Resolving